test/todo/FilterOptions.test.js

```diff
@@ -0,0 +1,23 @@
+import { shallowMount } from '@vue/test-utils'
+import FilterOptions from '@/components/todo/FilterOptions.vue'
+
+describe('FilterOptions.vue', () => {
```

Comment:
These APIs (describe, test, expect) seems to be unavailable in the test file. Please update jest config along with projects so they're available.

Reply:
![Screenshot 2023-01-23 at 11 11 02 AM](https://user-images.githubusercontent.com/85149436/213970114-28ac16f0-14fb-4552-b968-a1665ec10c3c.png)

Reply:
Resolved